pyproject.toml

```diff
@@ -3,4 +3,22 @@ requires = [
     "setuptools >= 47",
     "wheel"
 ]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
```

Comment:
should this be either left out or changed to 7.1.2?

Reply:
pytest only supports config in pyproject.toml at a min of 6.0
- https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml
I took this as a chance to upgrade our test dependencies (putting it to the latest one), since we hardly do it.

Reply:
Removed it.